go.mod

```diff
@@ -23,8 +23,8 @@ require (
 	github.com/IBM/ibm-hpcs-uko-sdk v0.0.20-beta
 	github.com/IBM/keyprotect-go-client v0.12.2
 	github.com/IBM/networking-go-sdk v0.45.0
-	github.com/IBM/platform-services-go-sdk v0.62.0
-	github.com/IBM/project-go-sdk v0.2.8-1
+	github.com/IBM/platform-services-go-sdk v0.62.3
+	github.com/IBM/project-go-sdk v0.2.1
```

Comment:
we are changing github.com/IBM/project-go-sdk verison value 
This change is breaking the go build 
https://github.com/IBM-Cloud/terraform-provider-ibm/actions/runs/8690624159/job/23917984722?pr=5270